loadJsonFile.py

```diff
@@ -0,0 +1,60 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# Load file -> load 'result.json'
+with open('result.json') as f:
+    read_data = f.read()
+
+# Dict type is 'list'
+datainfo = json.loads(read_data)
+
+data = dict()
+
+# If the information is unknown, save 'unknown'
+for item in datainfo:
+    try: data['name'] = item['name']
```

Comment:
'name' is used multiple times. It's not recommended to use the same hardcoded strings here and there. Can you create a variable and use that variable? e.g. name ='name', data[name]
Same with other variables